Classes/ViewHelpers/Security/KitodoRoleViewHelper.php

```diff
@@ -0,0 +1,36 @@
+<?php
+namespace EWW\Dpf\ViewHelpers\Security;
+
+/*
+ * This file is part of the TYPO3 CMS project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ *
+ * The TYPO3 project - inspiring people to share!
+ */
+
+class KitodoRoleViewHelper extends \TYPO3\CMS\Fluid\Core\ViewHelper\AbstractViewHelper
```

Comment:
What is special about "KitodoRoles" compared to just "Roles"?

Reply:
In TYPO3 role and group are used synonymously and to make it clear that it is not a TYPO3 / Fluid role, like in the fluid viewhelper "f: security.if HasRole", the naming "KitodRole" is used. A KitodoRole can be added to a fe user group to specify it as a kitodo role group.